django_python3_ldap/ldap.py

```diff
@@ -38,7 +38,11 @@ def _get_or_create_user(self, user_data):
         If the user does not exist, then it will be created.
         """
         User = get_user_model()
-        attributes = user_data["attributes"]
+
+        attributes = user_data.get("attributes")
+        if attributes is None:
+            return None
```

Comment:
There are other bits of the code that aren't expecting this function to return `None`. I think that you need to update `iter_users` to filter out users that are `None`.